### PR TITLE
c skaffold

### PR DESCRIPTION
**Pull request by smebberson**
This PR adds a new `c skaffold` command to make working with Skaffold integrate more easily with the conventions required when using Idearium cli.

## Related PRs

- idearium/ras-api#1

## Notes

- The `c kc build` command has also been updated to accept a `-t` option.
- This PR also adds prettier.

### Verification

N/A

### Testing

- [x] Pull down this project.
- [x] Execute `yarn` to install project dependencies.
- [x] Execute `yarn link`.
- [x] Go into a project using `c` and execute `yarn link @idearium/cli`.
- [x] Build a location with `c kc build <location>`, it should build.
- [x] Build all locations with `c kc build`, they should all build.
- [x] Build and deploy a location with `c kc build -d <location>`, it should build and deploy.
- [x] Test the setup process in the related PR, it should all work.

## Clean up

- [x] Execute `yarn unlink @idearium/cli` to remove the symlink.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -13,4 +13,4 @@ indent_size = 2
 
 # Because the `npm` cli resets it when you use `--save` or `--save-dev`.
```
Comment (allanchau) [inline .editorconfig]: Can this comment be removed now?

File: .eslintrc [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
```
Comment (allanchau) [inline .eslintrc]: Using `.eslintrc` without the extension has been deprecated, we should probably update this to include `.eslintrc.json` https://eslint.org/docs/user-guide/configuring#configuration-file-formats

File: bin/c-kc-build.js [hunk 3]
```diff
@@ -4,93 +4,109 @@
...
-        });
-
-    })
-    // eslint-disable-next-line handle-callback-err, no-unused-vars, no-empty-function
-    .catch((err) => {});
+            });
+        })
+        // eslint-disable-next-line handle-callback-err, no-unused-vars, no-empty-function
+        .catch((err) => {
+            console.error('ERRRRRRR', err);
```
Comment (allanchau) [inline bin/c-kc-build.js]: lol should we update this?

**Comment by smebberson** [comment]
@allanchau, I've tidied this up.

**Pull request merged**